tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -974,6 +1036,25 @@ private function getDatasetsForCustomizations(): array
         return $datasets;
     }
 
+    /**
+     * @param array $product
+     *
+     * @return ProductImage[]
+     */
+    private function createProductImages(array $product): array
```

Comment:
Once the image is handled by the pack query handle you can remove this one as well

Reply:
removed